x-pack/plugins/maps/public/components/layer_panel/flyout_footer/view.js

```diff
@@ -13,6 +13,10 @@ import {
   EuiSpacer,
   EuiButtonEmpty,
 } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
```

Comment:
Forgot to i18n the following
* `Cancel`, `Cancel` button text
* `Save &amp; close` button text